pkg/operator/status.go

```diff
@@ -96,28 +96,56 @@ func profileApplied(profile *tunedv1.Profile) bool {
 	return false
 }
 
-// profileDegraded returns true if Tuned Profile 'profile' has not been applied
-// or applied with errors (Degraded).
+// profileDegraded returns true if TuneD Profile 'profile' is Degraded
+// The Degraded ProfileStatusCondition occurs when a TuneD reports errors applying
+// the profile or when there is a timeout waiting for the profile to be applied.
 func profileDegraded(profile *tunedv1.Profile) bool {
 	if profile == nil {
 		return false
 	}
 
 	for _, sc := range profile.Status.Conditions {
-		if (sc.Type == tunedv1.TunedProfileApplied && sc.Status == corev1.ConditionFalse) ||
-			(sc.Type == tunedv1.TunedDegraded && sc.Status == corev1.ConditionTrue) {
+		if sc.Type == tunedv1.TunedDegraded && sc.Status == corev1.ConditionTrue {
 			return true
 		}
 	}
 
 	return false
 }
 
-// anyProfileDegraded returns true if any of the Tuned Profiles in the slice
-// 'profileList' has not been applied or applied with errors (Degraded).
+// profileNotApplied returns true if TuneD Profile 'profile' has not been applied
+// Returns true if the applied condition is False or Unknown.
+func profileNotApplied(profile *tunedv1.Profile) bool {
+	if profile == nil {
+		return false
+	}
+
+	for _, sc := range profile.Status.Conditions {
+		if sc.Type == tunedv1.TunedProfileApplied && sc.Status != corev1.ConditionTrue {
+			return true
+		}
+	}
+
+	return false
+}
+
+// anyProfileDegraded returns true if any of the TuneD Profiles in the slice
+// 'profileList' has not been applied or is Degraded.
 func anyProfileDegraded(profileList []*tunedv1.Profile) bool {
 	for _, profile := range profileList {
-		if profileDegraded(profile) {
+		if profileDegraded(profile) || profileNotApplied(profile) {
+			return true
+		}
+	}
+
+	return false
+}
+
+// anyProfileProgressing returns true if any of the TuneD Profiles in the slice
```

Comment:
Thanks for the context. In this case (and the prior function comment above) we can just say Profiles, instead of Tune(d/D) Profiles, since we are specifically referring to the Profile API object.